tools/depends/configure.ac

```diff
@@ -83,6 +83,11 @@ AC_ARG_WITH([sdk],
   [specify sdk platform version (optional for android). default is android-21])],
   [use_sdk=$withval])
 
+AC_ARG_WITH([ndk],
```

Comment:
OTOH it's now symetric with the sdk one.
If you want, we can have ndk-api and sdk-api, which makes sense.